react/features/welcome/components/WelcomePageVideoSwitch.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import React, { Component } from 'react';
+import {
+    Switch,
+    TouchableWithoutFeedback,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import { translate } from '../../base/i18n';
+import { updateProfile } from '../../base/profile';
+import { Header, Text } from '../../base/react';
+
+import styles, {
+    SWITCH_THUMB_COLOR,
+    SWITCH_UNDER_COLOR
+} from './styles';
+
+type Props = {
+
+    /**
+     * The Redux dispatch functions.
+     */
+    dispatch: Function,
+
+    /**
+     * The i18n translate function.
+     */
+    t: Function,
+
+    /**
+     * The current profile settings from Redux.
+     */
+    _profile: Object
+};
+
+/**
+ * Renders the audio-video switch on the welcome screen.
+ */
+class WelcomePageVideoSwitch extends Component<Props> {
```

Comment:
we are already in the welcome page feature, this can be called VideoSwitch.

Reply:
Sure thing!